_layouts/post.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="{{ site.lang | default: "en-UK" }}">
+        {% seo %}
+        {% include head.html %}
+  <body>
+    {% include navbar.html %}
+    <article class="post">
+        <div>
+            <h1 class="display-5">{{ page.title }}</h1>
+            <hr>
+            <div class="author d-flex align-items-center">
+                <div class="author_img">
+                  <img src="/assets/images/blog/author.png" alt="Author picture">
+                </div>
+                <div class="d-flex flex-column">
+                  <div class="author_name">{{page.author}}</div>
+                  <div class="article-date">{{ page.date | date: "%d.%m.%Y" }}
+                </div>
+                </div>
+            </div>
+            <hr>
+        </div>
+
+        <div class="post_body">
+            {{ content }}
```

Comment:
Add <div class="content-constrained"></div> wrapper around content to use the default container max-width defined in the other basic pages (About Us, etc.).

Reply:
Hm related to that, we should probably create a GitHub issue for creating a 'basic-page' layout file for the basic pages as well (About Us, etc.) - what do you think?

Reply:
Didn't use that because I wanted to specify a new max-width for individual posts. The other default size is too big so it extends all the way across the screen and doesn't make the article look good for reading. 
1024px max width looked better, in line with the design too.

Reply:
Oh I thought the content-constrained class made the max-width 768px and not full-width?

It should be consistent since it's just text content like basic pages.
- I was planning on creating an issue to adjust the global 'content-constrained' styles, because the characters per line is too long in terms of readability. UX-wise, typically between 50 to 75 characters, including spaces is recommended. Thus, it would be good to use the same wrapper class for these 'text-heavy' containers so the before-mentioned UX fix applies globally.

Reply:
So the fix the UX issue, either the container width needs to be decreased or we need to increase the base font size.

Reply:
Good point about the design, sorry that I haven't fixed the width in the Figma files. Will do

Reply:
I was referring to the default container-max-width for the main pages. For the content-constrained class, it looked too small and looked cramped. There was no in-between 😅

Reply:
Ah gotcha :) Please add the content-constrained. As mentioned I will create a separate issue for fixing the styles so we follow UX best practices and improve readability by displaying between 50 to 75 characters per line. Thanks

Reply:
And thank you for your quick reply to all my comments. Fyi, I've just started working on creating all components/pages from the site in Figma so that the design system and UX specs are more clear going forward :) A lot of documentation and code refactoring is pending lol due to the rapid development speed before launch :P